config/jobs/kubernetes/sig-windows/windows-gce.yaml

```diff
@@ -92,7 +92,7 @@ periodics:
   annotations:
     fork-per-release: "true"
     fork-per-release-replacements: "--extract=ci/latest -> --extract=ci/latest-{{.Version}}"
-    testgrid-dashboards: google-windows, sig-windows-gce, sig-windows-releases, sig-release-master-informing
+    testgrid-dashboards: google-windows, sig-windows-gce, sig-release-master-informing, sig-windows-signal
```

Comment:
Do we want to move the GCE configs into the ` config/jobs/kubernetes-sigs/sig-windows` folder? I think this is the folder where sig jobs are prefered to live.

Reply:
@jeremyje @pjh Any thoughts on if these should be consolidated?